### PR TITLE
Add recipe for blondies

### LINKED ISSUE #2: Problem: do muffins count?
There is a rather excellent and easy muffin recipe provided by @sallain 's grandmother but I'm concerned that muffins might be seen as scope creep?

### PR DESCRIPTION
**Pull request by sallain**
This is currently a work in progress. Will update when testing is complete.

Resolves #2

### CODE REVIEW

**Comment by sallain** [comment]
Tests definitely pass. Probably could've used 5 more minutes of baking time but I think that's an issue with my local setup rather than the instructions.

**Comment by Diogenesoftoronto** [comment]
Do you have any samples available for me to test?

**Comment by sallain** [comment]
@Diogenesoftoronto I don't think my samples will travel but it's super easy to build your own locally, only one bowl required.

File: blondies.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+- [ ]  1 large egg
+- [ ]  1 teaspoon vanilla extract
+- [ ]  1/2 teaspoon flaky sea salt, or 1/4 teaspoon coarse sea salt
+- [ ]  1 cup (130 grams) all-purpose flour
+- [ ]  1 1/2 cups mix-ins of you choice (my favorite: 1 cup chocolate chips plus
+  1/2 cup chopped toasted walnuts, see below for more suggestions)
+
+## Method
+
+1. Heat oven to 350°F. Butter an 8×8-inch baking pan, or line the bottom and two
```
Comment (eviau-artefactual) [inline blondies.md]: For clarity.

```suggestion
1. Heat oven to 350°F (around 175°C). Butter an 8×8-inch (20cm x 20cm) baking pan, or line the bottom and two
```

**Pull request merged**